packages/vuetify/src/components/VTabs/VTabs.ts

```diff
@@ -0,0 +1,292 @@
+      window.setTimeout(this.callSlider, 30)
+    })
+  },
+
+  methods: {
+    callSlider () {
+      if (
+        this.hideSlider ||
+        !this.$refs.items ||
+        !this.$refs.items.selectedItems.length
+      ) {
+        this.slider.width = 0
+        return false
+      }
+
+      this.$nextTick(() => {
+        // Give screen time to paint
+        const activeTab = this.$refs.items.selectedItems[0]
+        /* istanbul ignore if */
+        if (!activeTab || !activeTab.$el) {
+          this.slider.width = 0
+          this.slider.left = 0
+          return
+        }
+        const el = activeTab.$el as HTMLElement
+
+        this.slider = {
+          height: this.vertical ? el.offsetHeight : 2,
+          left: this.vertical ? 0 : el.offsetLeft,
+          right: this.vertical ? 0 : el.offsetLeft + el.offsetWidth,
+          top: el.offsetTop,
+          width: this.vertical ? 2 : el.scrollWidth
+        }
+      })
+
+      return true
+    },
+    genBar (items: VNode[], slider: VNode[]) {
+      return this.$createElement(VTabsBar, this.setTextColor(this.color, {
+        staticClass: this.backgroundColor,
+        style: {
+          height: this.height ? {
+            height: convertToUnit(this.height)
+          } : null
+        },
+        props: {
+          activeClass: this.activeClass,
+          // TODO: deprecate name
+          appendIcon: this.nextIcon,
+          dark: this.dark,
+          light: this.light,
+          // TODO: deprecate name
+          prependIcon: this.prevIcon,
+          mandatory: !this.optional,
+          mobileBreakPoint: this.mobileBreakPoint,
+          showArrows: this.showArrows,
+          value: this.internalValue
+        },
+        on: {
+          'call:slider': this.callSlider,
+          change: (val: any) => {
+            this.internalValue = val
+          }
+        },
+        ref: 'items'
+      }), [
+        this.genSlider(slider),
+        items
+      ])
+    },
+    genItems (items: VNode[], item: VNode[]) {
+      if (items.length > 0) return items
+      if (!item.length) return null
+
+      return this.$createElement(VTabsItems, {
+        props: {
+          value: this.internalValue
+        },
+        on: {
+          change: (val: any) => {
+            this.internalValue = val
+          }
+        }
+      }, item)
+    },
+    genSlider (items: VNode[]) {
+      if (this.hideSlider) return null
+
+      if (!items.length) {
+        const slider = this.$createElement(VTabsSlider, {
+          props: { color: this.sliderColor }
+        })
+
+        items = [slider]
+      }
+
+      return this.$createElement('div', {
+        staticClass: 'v-tabs__slider-wrapper',
+        style: this.sliderStyles
+      }, items)
+    },
+    onResize () {
+      if (this._isDestroyed) return
+
+      clearTimeout(this.resizeTimeout)
+      this.resizeTimeout = window.setTimeout(this.callSlider, 0)
+    },
+    parseNodes () {
+      const item = []
```

Comment:
Same as method above, having both item and items be arrays is counter-intuitive.